CppCoreGuidelines.md

```diff
@@ -5812,7 +5812,7 @@ But what if you can get significantly better performance by not making a tempora
         }
```

Comment:
might as well add the semicolon after` return *this` on the line just above

Reply:
Done